packages/core/src/lib/lit-core.ts

```diff
@@ -492,11 +507,6 @@ export class LitCore {
   }
 
   private async _connect() {
-    // Ensure an ill-timed epoch change event doesn't trigger concurrent config changes while we're already doing that
```

Comment:
Removed this call to stop listening -- if the code inside our listener calls `connect()`, it will just chain on any already-pending `connect()` logic (see `connect()` logic and promise chain on `this._connectingPromise.

Reply:
But will it chain the promises since we specifically check that if a pending connection is open then just return it?

Reply:
We always re-set `_connectionPromise` to null when a `connect()` call finishes -- so if we are connecting, multiple calls to `connect()` will all return the same promise.  This means that if we're still processing `connect()` when we receive another epoch change event, it will be effectively a no-op.

It occurred to me that we could implement cancellation across this entire callstack, and then cancel the entire existing call to connect() (along with any pending fetch() calls that it is running), and then run it again from the top. I'd like to implement a much more robust network handling layer based on the `v7` branch and land this as-is for now, since it's an improvement over what we've got.